thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/anomaly/task/TaskDriverConfiguration.java

```diff
@@ -25,6 +25,7 @@
   private int randomDelayCapInMillis = 15_000; // 15 seconds
   private int taskFetchSizeCap = 50;
   private int maxParallelTasks = 5;
+  private long maxTaskRunTimeMillis = 6 * 60 * 60_000; // 6 hours
```

Comment:
This should be final.

Reply:
We probably don't want it to be final, as this is just default value. It can be set via the setter if needed in the future.

Reply:
Sounds good.